assets/vsphere_problem_detector/12_prometheusrules.yaml

```diff
@@ -17,12 +17,16 @@ spec:
         labels:
           severity: warning
         annotations:
-          summary: "VSphere node health checks are failing"
+          summary: |
+            Periodic vSphere health check {{ $labels.check }} is failing on node {{ $labels.node }}.
+            To get details about the failure, please see events in namespace openshift-cluster-storage-operator:
+            'oc -n openshift-cluster-storage-operator get event --sort-by=.metadata.creationTimestamp | grep VSphere{{ $labels.check }}'
+          message: "VSphere health check {{ $labels.check }} is failing on {{ $labels.node }}."
```

Comment:
nit: technically the correct is `vSphere`

Reply:
Fixed